test/snippet/search/dream_index/counting_agent.cpp

```diff
@@ -0,0 +1,46 @@
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/core/debug_stream.hpp>
+#include <seqan3/range/views/kmer_hash.hpp>
+#include <seqan3/search/dream_index/interleaved_bloom_filter.hpp>
+
+using seqan3::operator""_dna4;
+
+int main()
+{
+    seqan3::interleaved_bloom_filter ibf{seqan3::bin_count{8u},
+                                         seqan3::bin_size{8192u},
+                                         seqan3::hash_function_count{2u}};
+
+    auto const sequence_1 = "ACTGACTGACTGATC"_dna4;
+    auto const sequence_2 = "GTGACTGACTGACTCG"_dna4;
+    auto const sequence_3 = "AAAAAAACGATCGACA"_dna4;
```

Comment:
```suggestion
    auto const sequence1 = "ACTGACTGACTGATC"_dna4;
    auto const sequence2 = "GTGACTGACTGACTCG"_dna4;
    auto const sequence3 = "AAAAAAACGATCGACA"_dna4;
```

No hard-feelings for this change, but I think we normally don't underscore before numbers.